balls.cpp

```diff
@@ -1,7 +1,9 @@
 // BALLS
-#include <iostream>
+#ifndef _BALLS
+#define _BALLS
+#include <iostream> 
 
 int main() {
-  std::cout << "balls" << std::endl;
-  return 0;
+  std::cout << "balls\n";
```

Comment:
At the end of the day, we only flush the buffer once so does it matter that much?

Reply:
Okay, I tested this and there is an extremely minor performance improvement using the line break character instead of std::endl